### PR TITLE
To do list

### PR DESCRIPTION
**Pull request by dev1980**
>Created Todo lists app with OOP principles.
>We use HTML, CSS, JS, and Webpack.
>We add features to create Projects and add a task to each project.

Implement all the required changes:
> Create a new todo-dom file to accommodate the TodoDom factory function for the renderTodo method.
> Set the default project.
> Update the readme file.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    const projectObj = new Project(projectName.value);
+    todoProject.push(projectObj);
+    updateLocalStorage(todoProject);
+    projectList.appendChild(project);
+    currentProject = getProject(parseInt(project.getAttribute('data-index'), 10));
+    projectName.value = '';
+  }
+}
+
+function renderTodo(currProject) {
```
Comment (bellom) [inline src/index.js]: Let us move this function out.
image
https://www.theodinproject.com/courses/javascript/lessons/todo-list

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
Below are part of the checklist for this project, kindly work on these few things listed below;

1. There should be a  'default' project`
image
https://www.theodinproject.com/courses/javascript/lessons/todo-list

2. **README file, not detailed**. (you can add steps to git clone your repo, necessary installation to setup project locally and live link)

Kindly watch the video tutorial on how to create a live link 👉  [Link](https://www.loom.com/share/76088842a2d14322bbcf65b884f4d42d);

checkout out the below link for help, free free to follow the template from the pathwright
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/


**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED 

Hi Team good job on this project everything works fine now, kudos on making previous suggested changes :+1:  just one thing left to do. I noticed you changed the eslint file
see this
[see correct eslint file](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) 

please make changes and request another review :+1:  

**RAPHAEL NORIODE** :sunglasses:

**Comment by lekegitrepo** [comment]
We didn't edit/change the eslint file we forget to copy and paste the required one from the linter-config repo

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**